src/application-status-overlay/ApplicationStatusOverlayProvider.module.scss

```diff
@@ -0,0 +1,20 @@
+:local {
+  .container {
+    height: 100%;
+    outline: none;
+    position: relative;
+  }
+
+  .container-content {
+    height: 100%;
+  }
+
+  .status-view {
+    background-color: #fff;
+    outline: none;
+    overflow: hidden;
+    position: absolute;
+    width: 100%;
+    z-index: 100;
```

Comment:
This seems arbitrarily high. I think you could put an explicit z-index of 0 on `.container`, 0 on the `.status-view`, and 1 on the `.content-container` and get guaranteed stacking order you're wanting.

Reply:
FYI anyone who wants to learn about z-index magic: https://philipwalton.com/articles/what-no-one-told-you-about-z-index/

Article is old as dirt but still accurate, as far as I can tell. Helped me visualize how some of this works.

Reply:
The intention for the high z-index value was to account for user defined z-index values for the children elements.
But i agree that adding z-index of 0 to the `div` wrapping the children (container-content) does seem to address my concern much better.
Addressed here : https://github.com/cerner/terra-application/pull/47/commits/5e92dd7f93fe585a4883c6aa4734cb2859e0fa06